docs/PokemonCreation.md

```diff
@@ -0,0 +1,70 @@
+
+A newly created Pokemon's member types can be found in [`Battles.ts`](../src/components/Battles.ts) under the IPokemon interface.
+
+### Equations
+
+`Equations.ts` contains the functions necessary for choosing a random Pokemon (`chooseRandomWildPokemon`), creating a new Pokemon (`newPokemon`), choosing that new Pokemon's statistics (`newPokemonStatistics`), and choosing a new Pokemon's IVs and EVs (`newPokemonIVs` and `newPokemonEvs`).
+
+### Utilities
+
+`Utilities.ts` contains the function necessary for calling `newPokemon` in `Equations.ts` (`createPokemon`), which uses the chosen random Pokemon, from `Equations.ts` `chooseRandomWildPokemon`, to get the get the level of the chosen Pokemon and title which it then feeds to `newPokemon`.
+
+### Unit Test and Trainer Battles
+
+Pass into the `newPokemon` function from `Equations.ts` an object with appropriate members.
+
+```javascript
+const pokemonTitle: string[] = "CHARMANDER".split("");
+const pokemonLevel: number = 10;
+const pokemon: IPokemon = FSP.equations.newPokemon({
+    level: pokemonLevel,
+    title: pokemonTitle
+});
+const pokemon: IPokemon = FSP.equations.newPokemon({
+    item: "Potion".split(""),
+    level: pokemonLevel,
+    moves: [{
+        title: "Ember",
+        remaining: FSP.constants.moves.Ember.pp,
+        uses: FSP.constants.moves.Ember.pp
+        }],
+    title: pokemonTitle
+});
+```
+
+### Wild Pokemon Encounter
+
+The steps for creating a Pokemon in a wild Pokemon encounter are:
+1. The `chooseWildPokemonForBattle` function is run.
+    * This function returns the Pokemon object by using the grass `IMap` and `IArea` to determine what wild Pokemon can be encountered.
+    * The grass `IMap` and `IArea` determine which wild Pokemon can be encountered by looking at the current location which is an object labeled `areas` that includes an object `wildPokemon` that holds what Pokemon can be found along with appearance rates.
+2. After selecting a valid wild Pokemon that can be encountered `chooseRandomWildPokemon` is called and chooses a Pokemon from the list of valid Pokemon based on the chance of encountering that wild Pokemon.
+3. Next, `createPokemon` is called which sets up an object of type `INewPokemon` containing information on the level and title of the chosen Pokemon.
+4. `newPokemon` is called creating and returning the wild encountered Pokemon.
+
+```javascript
+const options: IWildPokemonSchema[] = [
+    {
+        title: "CHARMANDER".split(""),
+        level: 10,
+        moves: [{
+            title: "Ember",
+            remaining: FSP.constants.moves.Ember.pp,
+            uses: FSP.constants.moves.Ember.pp
+            }],
+        rate: .25
+    },
+    {
+        title: "Squirtle".split(""),
+        levels: [10,5,4,7],
+        rate: .75
+    }
+];
+const chosen: IWildPokemonSchema = fsp.equations.chooseRandomWildPokemon(options);
```

Comment:
Can you put a comment above `chosen` saying that it's got a 75% chance of being a Squirtle and 25% of Charmander? Should help clarify how these interact.